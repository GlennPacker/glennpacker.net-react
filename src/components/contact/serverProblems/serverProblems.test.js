import React from 'react';
import { shallow } from 'enzyme';

import ServerProblems from './index';

describe('Server Problems', () => {
    let wrapper;

    beforeAll(() => {
        wrapper = shallow(
            <ServerProblems />
        );
    })

    it('should render matching snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})