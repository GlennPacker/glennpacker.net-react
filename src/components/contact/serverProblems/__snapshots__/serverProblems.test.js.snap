// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Server Problems should render matching snapshot 1`] = `
<div
  className="container pt-5"
>
  <div
    className="col-sm"
  >
    <h1>
      Yikes
    </h1>
    <p
      className="pt-3"
    >
      I would get back to you but the server is having problems.
    </p>
  </div>
</div>
`;
